app/components/listing/RelativeOpeningTime.jsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import moment from 'moment';
 
-// TODO Until we use actual timestamps, this one constant converts everything to PST below
-const timezoneOffset = 8;
+// TODO Until we use actual timestamps, this one constant converts everything to the correct timezone
+const timezoneOffset = new Date().getTimezoneOffset() / 60;
```

Comment:
@lexholden I think I fixed the Travis issues with this one change. The problem with Travis is that their servers set the timezone to UTC, so the hardcoded constant wasn't working correctly on their servers, even if they work on a browser located in California. This is probably a slightly better approach anyway, since San Francisco shifts to -7 hours during Daylight Savings.